test/functional/test_framework/authproxy.py

```diff
@@ -139,17 +139,24 @@ def _request(self, method, path, postdata):
             return self._get_response()
 
     def __call__(self, *args, **argsn):
-        AuthServiceProxy.__id_count += 1
+        # check for a possible wallet endpoint
+        urlAdd = ""
+        method = self._service_name
+        elements = self._service_name.split(".")
+        if len(elements) > 1:
+            urlAdd = "/wallet/" + '.'.join(elements[:-1])
```

Comment:
I agree - we should try to make minimal changes to authproxy.

Later, once TestNode is merged, it'll be very straightforward to add wallet methods to the TestNode class, which is where I think they should live - not in the authproxy layer.